package.json

```diff
@@ -49,7 +48,8 @@
     "eslint": "4.19.1",
     "eslint-config-airbnb-base": "12.1.0",
     "eslint-plugin-import": "2.12.0",
+    "git-hooks": "1.1.10",
```

Comment:
Looking at this PR https://github.com/ForestAdmin/lumber/pull/303
Shouldn't you do it for `simple-git` too?

Reply:
You are right simpleGit is for dev only.